svgdigitizer/__main__.py

```diff
@@ -4,16 +4,18 @@
 def cli(): pass
 
 @click.command()
+@click.option('--sampling_interval', type=float, default=None, help='specify sampling interval (for now in mV)')
 @click.argument('svg')
-def plot(svg):
+def plot(svg, sampling_interval):
     from svgdigitizer.svgdigitizer import SvgData
-    SvgData(svg).plot()
+    SvgData(svg, sampling_interval=sampling_interval).plot()
 
 @click.command()
+@click.option('--sampling_interval', type=float, default=None, help='specify sampling interval (for now in mV)')
```

Comment:
It's weird that we now have to specify all the parameters more than once.

Maybe, we should use global parameters to configure `SvgData`? I.e., maybe `CreateCVData` should be refactored so it is fed a `SvgData` object instead of taking any parameters itself?